docs/src/validator/best-practices/security.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Validator Security Best Practices
+sidebar_label: Security
+---
+
+Being a system administrator for an Ubuntu computer requires technical knowledge of the system and best security practices. The following list should help you get started and is considered the bare minimum for keeping your system safe.
+
+## Keep Your System Up To Date
+
+Make sure to regularly update packages in your Ubuntu system. Out of date packages may contain known security vulnerabilities that a hacker could exploit. A good practice would be to update weekly at least. To update your system, do the following:
+
+```
+sudo apt update
+sudo apt upgrade
+```
+
+## DO NOT Store Your Withdrawer Key On Your Validator Machine
```

Comment:
Should we strongly recommend validators use a ledger for the withdrawer?

Reply:
It's a good solution but I don't want to be vendor specific in the docs.

Reply:
Perhaps we can just generally recommend a cold wallet

Reply:
I'd maybe mention a few options, including paper wallet, Ledger or Keystone wallet or a Squads multisig backed by any combination of these.